OpenImis.ModulesV2/ClaimModule/Logic/ClaimLogic.cs

```diff
@@ -1,14 +1,27 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using Microsoft.Extensions.Configuration;
+using OpenImis.ModulesV2.ClaimModule.Models.RegisterClaim;
+using OpenImis.ModulesV2.ClaimModule.Repositories;
 
 namespace OpenImis.ModulesV2.ClaimModule.Logic
 {
     public class ClaimLogic : IClaimLogic
     {
-        public ClaimLogic()
+        private IConfiguration _configuration;
+        protected IClaimRepository claimRepository;
+
+        public ClaimLogic(IConfiguration configuration)
         {
+            _configuration = configuration;
+            claimRepository = new ClaimRepository(_configuration);
+        }
+
+        public int RegisterClaim(Claim claim)
```

Comment:
In FamilyLogic you are naming the method Create. To be consistent, can we use the same method names between controllers, logics and repositories? I suggest using the CRUD naming: create, read, update and delete to start with.

Reply:
@dragos-dobre  It's a good idea. I changed the name